src/ui/map.js

```diff
@@ -1891,6 +2057,9 @@ class Map extends Camera {
      * @param {Object} [options] Options object.
      * @param {boolean} [options.validate=true] Whether to check if the filter conforms to the Mapbox GL Style Specification. Disabling validation is a performance optimization that should only be used if you have previously validated the values you will be passing to this function.
      * @returns {Map} `this`
+     * @example
+     * const layerVisibility = map.getLayoutProperty('my-layer', 'visibility');
```

Comment:
Replace const to stay consistent
```suggestion
     * var layerVisibility = map.getLayoutProperty('my-layer', 'visibility');
```